### DISCUSSION TITLE
PyQt5

### DISCUSSION THREAD

**Discussion by matveygu** [community/community#109656] [Programming Help]
How to switch off tab(concentrate on buttons) in my window?

**Comment by alrnaldar** [comment]
If you want to disable the tab focus on your window and instead focus solely on buttons or other interactive elements, you can do so by modifying the window's settings. 
### Here's how you can achieve this in a Kivy application:
```
from kivy.app import App
from kivy.uix.button import Button
from kivy.config import Config

class MyApp(App):
    def build(self):
        # Create a button
        button = Button(text='Click Me!')
        return button

if __name__ == '__main__':
    # Disable tab focus
    Config.set('kivy', 'keyboard_mode', 'systemandmulti')

    # Run the application
    MyApp().run()
```
In this code:

Config.set('kivy', 'keyboard_mode', 'systemandmulti'): This line sets the keyboard mode to 'systemandmulti', which disables the tab focus and allows the system to handle the keyboard input.
By default, Kivy's keyboard mode is set to 'managed', which allows Kivy to handle keyboard input and enables tab focus. By setting it to 'systemandmulti', you disable Kivy's keyboard handling and allow the system to handle the keyboard input, effectively disabling tab focus within your Kivy window.

With this setup, when you run your Kivy application, tab focus will be disabled, and the focus will remain on buttons or other interactive elements without switching to other widgets when pressing the Tab key.

**Comment by Akash1134** [comment]
Hi @matveygu , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
